src/tests/Common/CLRTest.CrossGen.targets

```diff
@@ -69,95 +69,103 @@ fi
 
 # CrossGen2 Script
 if [ ! -z ${RunCrossGen2+x} ]%3B then
+  compilationDoneFlagFile="IL-CG2/done"
+  if [ -d IL-CG2 ]%3B then
+    while [ ! -f $compilationDoneFlagFile ]%3B
+    do 
+      echo "Waiting for concurrent Crossgen2 compilation: $compilationDoneFlagFile"
+      sleep 10%3B
```

Comment:
A nit - I was wondering if 10 seconds isn't unnecessarily too long, but I don't have a strong opinion on this.

Reply:
I don't have a strong opinion either, I reused the same value that Crossgen1 uses. Please note that, when 1000 tests targeting the same executable are run at exactly the same moment, the first one launches Crossgen2 and all the remaining 999 tests wait for 10 seconds and then start concurrently so the "unnecessary wait time" overhead should be only proportional to the number of test groups grouped by the targeting executable minus minimum CG2 running time which is realistically 1-2 seconds even for a small test.